tests/structuredheadings/clearnumberingstyle.js

```diff
@@ -1,5 +1,7 @@
+/* eslint-disable */
 /* bender-tags: editor,unit */
-/* bender-ckeditor-plugins: wysiwygarea,structuredheadings,toolbar,basicstyles,dialog,richcombo */
+/* bender-ckeditor-plugins: wysiwygarea,structuredheadings,toolbar,basicstyles,dialog,richcombo,undo */
+/* eslint-enable */
```

Comment:
it chokes on things in the bender comment, like the excessive line mostly.

I guess i could have just disabled the one rule in block form.  the same line ignore doesn't work because it's already a comment.